bigcode_eval/tasks/humaneval.py

```diff
@@ -13,6 +13,7 @@
 from evaluate import load
```

Comment:
we can remove the evaluate calls

Reply:
cleaned up - ok to merge if you're happy about it